apps/server/src/shared/domain/entity/external-tools/external-tool/external-tool.entity.spec.ts

```diff
@@ -35,20 +35,16 @@ describe('ExternalTool Entity', () => {
 				type: ToolConfigType.OAUTH2,
 				baseUrl: 'mockBaseUrl',
 				clientId: 'mockClientId',
-				clientSecret: 'mockClientSecret',
 				skipConsent: true,
-				frontchannelLogoutUrl: 'mockUrl',
 			});
 			const lti11ToolConfig: Lti11ToolConfig = new Lti11ToolConfig({
 				type: ToolConfigType.LTI11,
 				baseUrl: 'mockBaseUrl',
 				key: 'mockKey',
 				secret: 'mockSecret',
-				resource_link: 'mockLink',
+				resource_link_id: 'mockLink',
```

Comment:
Did you also renamed it in the entity and response? No one is using this responses.. so we have the chance to change the name

Reply:
This is the naming for resource link in the Lti11 Standard and yes i checked this.